gcn_email_service/__init__.py

```diff
@@ -63,10 +64,14 @@ def query_and_project_subscribers(table, topic):
 
 
 def connect_as_consumer():
-    return Consumer(config_from_env())
+    return Consumer(**config_from_env())
```

Comment:
Fixed in gcn-kafka:

```suggestion
```

But please update pyproject.toml to require `gcn-kafka = "^0.2.1"` to pick up this fix.